components/shader/shadermanager.cpp

```diff
@@ -172,7 +172,7 @@ namespace Shader
             size_t contentEnd = source.find("$endforeach", contentStart);
             if (contentEnd == std::string::npos)
             {
-                std::cerr << "Unexpected EOF" << std::endl;
+				Log(Debug::Error) << "Unexpected EOF";
```

Comment:
I think I have a bug to report to someone, because it should be literally impossible to put a tab into a C++ file in this project with the tools I'm using. That's the whole reason I added a `.editorconfig` file.